tests/performance/config/perf-driver/policies/test-1-apps-n-instances.yml

```diff
@@ -48,9 +48,7 @@ policies:
       - name: Scale Instances
         values:
           - parameter: instances
-            min: 50
-            step: 50
-            max: 500
+            values: [500, 1000, 2000]
```

Comment:
How about simply `values: [5000]`? ;)

Reply:
I would vote against this, for various reasons, but mainly:

1) Having (at least) 3 data points allows you to interpolate/extrapolate the numbers in other regions that you haven't collected data for.
2) Some performance issues show some tendency, and having at least 3 data points helps you identify them.
3) Having enough (ok, perhaps more than 3) data points, helps you detect outliers.

Nonetheless, with just 3 data points the run time has been decreased down to about 20 minutes, and most of it is just building marathon 😄 . I don't think we gain much by reducing this down to a single point...

_(Though I can make them `[1000, 3500, 6000]` if you like)_

Reply:
1. We do have multiple data points - these are the deployment time for individual deployments (or Instance-started events for 1app-Ninstances scenario) 
2. We will gather more data by running perf tests in the loop every 30m or so
3. We will gather more interesting data by sending percentiles to DD

Reply:
Coming back to what you want to do, you should check the #6300 PR instead.
I am going to insist that we have (at least) 3 data points in this PR, because it's intended for a different purpose than what (I think) you have in mind.